File: frontend/src/models/PointSale.ts
export interface PointSale {
  IdPointSale: number;
  codePointSale: string;
  namePointSale: string;
  statusPointSale: boolean;
}

export interface PointSaleCreate {
  codePointSale: string;
  namePointSale: string;
  statusPointSale: boolean;
}

export type PointSaleUpdate = Partial<PointSaleCreate>;

File: backend/app/domain/entities/pointSale.py
from sqlalchemy import Column, Integer, String, Boolean
from app.infrastructure.db.connection import Base

class pointSale(Base):
    __tablename__ = "pointSale"

    IdPointSale = Column(Integer, primary_key=True, index=True, autoincrement=True)
    codePointSale = Column(String(50), nullable=False, unique=True)
    namePointSale = Column(String(150), nullable=False)
    statusPointSale = Column(Boolean, nullable=False, default=True)